core/trino-main/src/main/java/io/trino/sql/planner/planprinter/anonymize/Anonymizer.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.sql.planner.planprinter.anonymize;
+
+import io.trino.execution.TableInfo;
+import io.trino.metadata.IndexHandle;
+import io.trino.metadata.QualifiedObjectName;
+import io.trino.metadata.TableExecuteHandle;
+import io.trino.metadata.TableHandle;
+import io.trino.spi.connector.ColumnHandle;
+import io.trino.spi.type.Type;
+import io.trino.sql.planner.PartitioningHandle;
+import io.trino.sql.planner.Symbol;
+import io.trino.sql.tree.Expression;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.trino.sql.planner.Partitioning.ArgumentBinding;
+import static io.trino.sql.planner.plan.StatisticsWriterNode.WriteStatisticsTarget;
+import static io.trino.sql.planner.plan.TableWriterNode.WriterTarget;
+
+public interface Anonymizer
```

Comment:
One implementation performs anonymisation, the other doesn't. Could you take a look at its usage in PlanPrinter and suggest a different way if this can be improved ?